### PR TITLE
Update Danish Translation (+70% / 796)

### PR DESCRIPTION
**Pull request by Stisen1**
- Focused translations on advendture map object prompts
- Some changes to creature and race names and to lines that has these new names in their contexts

### CODE REVIEW

File: files/lang/dk.po [hunk 1]
```diff
@@ -3382,15 +3379,13 @@ msgstr "Personlighed:"
 msgid "Best Monster:"
 msgstr "Bedste væsen:"
 
-#, fuzzy
 msgid ""
 "Are you sure you want to load a new map? (Any unsaved changes to the current "
 "map will be lost.)"
 msgstr ""
 "Er du sikker på, om du vil indlæse et nyt spil? (Dit nuværende spil vil gå "
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:3386]: I tried to change these old errors beforehand, but some lines went under the radar... It has been corrected.

File: files/lang/dk.po [hunk 2]
```diff
@@ -3382,15 +3379,13 @@ msgstr "Personlighed:"
...
 "tabt.)"
 
-#, fuzzy
 msgid ""
 "Are you sure you want to create a new map? (Any unsaved changes to the "
 "current map will be lost.)"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:3391]: It has been corrected.

File: files/lang/dk.po [hunk 3]
```diff
@@ -4671,10 +4651,10 @@ msgid "Click to return to the main menu."
 msgstr "Klik her for at vende tilbage til hovedmenuen."
 
 msgid "Astrologers proclaim the Month of the %{name}."
-msgstr "Astrologer kundgør, at det er måneden for %{name}."
+msgstr "Astrologer kundgør, at månedens væsen er %{name}."
 
 msgid "Astrologers proclaim the Week of the %{name}."
-msgstr "Astrologer kundgør, at det er ugen for %{name}."
+msgstr "Astrologer kundgør, at ugens væsen er %{name}."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:4657]: It says "... at ugens væsen er FORPESTET", when the "week of PLAGUE" occurs.

File: files/lang/dk.po [hunk 4]
```diff
@@ -5352,83 +5334,103 @@ msgstr ""
...
 msgid "You come upon the remains of an unfortunate adventurer."
-msgstr ""
+msgstr "Du støder på resterne af en uheldig eventyrer."
 
 msgid "Treasure"
 msgstr "Skat"
 
 msgid "Searching through the tattered clothing, you find the %{artifact}."
-msgstr ""
+msgstr "Du gennemsøger det lasede tøj, finder du %{artefakten}."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:5343]: Darn Google.Translate... It has been corrected.

File: files/lang/dk.po [hunk 5]
```diff
@@ -5352,83 +5334,103 @@ msgstr ""
...
+"Du støder på en gammel vogn efterladt af en handelsmand, der ikke helt nåede "
+"til sikkert terræn."
 
 msgid "Unfortunately, others have found it first, and the wagon is empty."
-msgstr ""
+msgstr "Desværre har andre fundet den først, og vognen er tom."
 
 msgid "Searching inside, you find the %{artifact}."
-msgstr ""
+msgstr "Indenfor kigger du søgende rundt. Du finder %{artefakten}."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:5359]: It has been corrected.

File: files/lang/dk.po [hunk 6]
```diff
@@ -6614,63 +6617,64 @@ msgstr ""
 "\"På mine rejser har jeg lært meget om mystisk magi. Et stort orakel lærte "
 "mig sine færdigheder. Jeg har svaret, du søger.\""
 
-#, fuzzy
 msgid "No spell book is present."
 msgstr "Ingen magi at kaste."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:6621]: Wierd. I could have sworn I've already corrected this ones before...

File: files/lang/dk.po [hunk 7]
```diff
@@ -7803,22 +7805,22 @@ msgid "Witch's Hut"
 msgstr "Heksens hytte"
 
 msgid "Xanadu"
-msgstr ""
+msgstr "Abrahams skød"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:7808]: I've never heard of anything called Xanadu, so I wanted to call it something that danish gamers would have an easier chance recognizing. But it has been changed to Xanado. Apparently some danish garden company has this name.

File: files/lang/dk.po [hunk 8]
```diff
@@ -7360,8 +7362,8 @@ msgstr ""
 
 msgid "doubleLined|Wizard"
 msgstr ""
-"Trold-\n"
-"mand"
+"Magi-\n"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:7365]: Apparently it doesn't. I've just checked in-game. It has been corrected.

**Comment by ihhub** [comment]
@Stisen1 , thank you so much for these constant updates!

**Pull request merged**